### PR TITLE
Prettier workflow names

### PR DESCRIPTION
**Pull request by aaronj1335**
Extracted from discussion in #25.

### CODE REVIEW

**Comment by aaronj1335** [comment]
For reference, here's how this looks in the `checks` tab https://github.com/aaronj1335/gmail-automata/pull/4/checks

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -10,13 +9,28 @@ on:
...
-    - uses: actions/setup-node@v1
-    - run: npm ci
-    - run: npm run build --if-present
-    - run: npm test
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Setup Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.x'
```
Comment (ranmocy) [inline .github/workflows/node.js.yml:21]: Like Travis config, could we always use the latest version?

**Comment by aaronj1335** [comment]
Looking at the docs and code for setup node, I did not see a way to do
that. I Yes it kind of makes sense in a way, because usually you don't want
CI to ever break if you didn't change anything. But I also understand how
since we don't control the deploy target there is an argument for it.

I am also not sure how node is getting set up without this directive. It
seems like we are relying on it just being present in whatever container
this is running on.

**Comment by ranmocy** [comment]
I couldn't find it's officially supported neither. Let's just keep this for now.

I'm not sure what you mean by setting up nodejs without this directive. It's surely required and it was in the config.

**Comment by aaronj1335** [comment]
Ah right, it was... Weird. I guess we could just remove the version directive and see what happens 🤷‍♂️

**Pull request merged**